src/components/UserList/index.tsx

```diff
@@ -62,13 +62,22 @@ const messages = defineMessages({
     'Email notifications need to be configured and enabled in order to automatically generate passwords.',
   autogeneratepassword: 'Automatically generate password',
   validationEmail: 'You must provide a valid email address',
+  sortCreated: 'Creation Date',
+  sortUpdated: 'Last Updated',
+  sortUsername: 'Username',
```

Comment:
Should be "Display Name" and we should probably rename the table header as well.

Reply:
I thought that too, but I figured we should be consistent with the table header.  Changing this now :)